docs/language.js

```diff
@@ -397,7 +397,20 @@ var data = [
 		"description" : "minimum of all of the numbers",
 		"example" : "(print (min 0.5 1 7 9 -5))"
 	},
-
+	{
+		"parameter" : "index_max [number x1] [number x2] [number x3] ... [number xN] | (assoc ...) | (list ...)",
+		"output" : "*",
+		"concurrency" : true,
+		"description" : "the index of of the argument with the greatest value, or the key associated with the greatest value in an assoc, or the index of the greatest value in a list",
```

Comment:
Should have looked into this PR earlier, sorry. I think I had put it in the ticket for this, but index_max/index_min need to return the *list* of indices that map to the maximum or minimum value.

This way the user is responsible for deciding the logic of which index to "get" in the case of ties within the structure. For example, the use-case that prompted me to ask for this would always desire the first index with the max value, but I can imagine other cases where I may want the last. I don't think these opcodes should only return one of the indices with the extreme value.

Many libraries only return the first, but in the case of (assoc)'s, the order is a bit hard to predict anyway.

Reply:
Done.